plugin/nginx_plus_collectd.py

```diff
@@ -281,6 +283,8 @@ def configure(self, conf):
         status_port = None
         username = None
         password = None
+        legacy_api = None
+        api_version = None
```

Comment:
Would it make more sense to treat `api_version` as `None` to be legacy instead of having two variables?  We could set the default of 3 here.

Reply:
Hello @rmfitzpatrick,

As per the latest design(explained [here](https://github.com/signalfx/collectd-nginx-plus/pull/8#discussion_r757689243)), we are determining `legacy` or `newer` APIs by a single config option of `APIVersion` and have removed the `LegacyAPI` config option. 

Please let us know in case of any further changes required.